web_src/js/features/common-global.js

```diff
@@ -101,17 +104,20 @@ export function initGlobalCommon() {
     },
     onHide() {
       this._tippy?.enable();
+
+      // hide all tippy elements of items after a while. eg: keyboard Enter to trigger the "Copy Link" in the Issue Context Menu
+      setTimeout(() => {
+        $(this).find('.menu > .item').each((_, item) => {
+          item._tippy?.hide();
+        });
+      }, 2000);
```

Comment:
This seems like a hack. What if the user re-hovers the button within those two seconds? It will disruptingly hide the tooltip.

Reply:
I can add a `is visible` check.

Reply:
Done https://github.com/go-gitea/gitea/pull/23450/commits/15285595298994035f4d5e0aba1c44d026c4550b

Reply:
Guess that is better, but I'd still prefer if we can somehow do this without a `setTimeout`.

Reply:
I didn't find an approach. Could you help to try and suggestion an approach?